resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,10973 @@
+                </tr>
+                <tr>
+                    <td><p>Request containing invalid and/or missing data</p><p class ="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a></p></td>
+                    <td>
+        <pre class="code--block">
+        {
+          "investorId": "9876543",
+          "creationReason": "New",
+          "firstSubscriptionDate": "2011"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">400 (Bad Request)</code></p>
+        <pre class="code--block">
+        {
+          "code": "BAD_REQUEST",
+          "message": "Bad Request",
+          "errors": [
+            {
+              "code": "INVALID_DATE",
+              "message": "Date is invalid",
+              "path": "/firstSubscriptionDate"
+            },
+            {
+              "code": "INVALID_FORMAT",
+              "message": "Invalid format has been used",
+              "path": "/investorId"
+            },
+            {
+              "code": "MISSING_FIELD",
+              "message": "This field is required",
+              "path": "/accountId"
+            }
+          ]
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td><p>Request containing dates before 6 April 2017</p><p class ="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a></p></td>
+                    <td>
+        <pre class="code--block">
+        {
+          "investorId": "9876543210",
+          "creationReason": "Transferred",
+          "accountId": "1234567890",
+          "firstSubscriptionDate": "2016-04-06",
+          "transferAccount": {
+            "transferredFromAccountId": "8765432100",
+            "transferredFromLMRN": "Z654321",
+            "transferInDate": "2016-04-06"
+          }
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">403 (Forbidden)</code></p>
+        <pre class="code--block">
+        {
+          "code": "FORBIDDEN",
+          "message": "There is a problem with the request data",
+          "errors": [
+            {
+              "code": "INVALID_DATE",
+              "message": "The firstSubscriptionDate cannot be before 6 April 2017",
+              "path": "/firstSubscriptionDate"
+            },
+            {
+              "code": "INVALID_DATE",
+              "message": "The transferInDate cannot be before 6 April 2017",
+              "path": "/transferAccount/transferInDate"
+            }
+          ]
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td><p>Request containing investor details which cannot be found</p><p class ="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a></p></td>
```

Comment:
Once the duplicated Testing section in this file is removed, please update link from`#section/Testing` to `/api-documentation/docs/api/service/lisa-api/2.0#testing`. Thanks